src/credentials/CHIPCert_Internal.h

```diff
@@ -23,10 +23,17 @@
 namespace chip {
 namespace Credentials {
 
+// The decode buffer is used to reconstruct TBS section of X.509 certificate, which doesn't include signature.
+inline constexpr size_t kMaxCHIPCertDecodeBufLength = kMaxDERCertLength - Crypto::kMax_ECDSA_Signature_Length_Der;
```

Comment:
I recommend against this heuristic, since it tries to save ~70 bytes of heap/stack in a few places with a very rough low-level assumption. I recommend using kMaxDERCertLength which is guanrateed to fit for all chip certs.

Reply:
Yeah I just moved this from the public header into the internal one since this doesn’t seem like it’d API (and there is an internal header now)